pages/how-to/set-up/scm-connection.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Setting Up an SCM Connection
+description: Connecting Plural to a Source Control Management Provider
+---
+
+# Prerequisites
+* **Plural Console `admin` Permissions**  
+* **SCM Provider Personal Access Token**
+  * [Github](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)  
+  * [GitLab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token)  
+  * [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/access-tokens/)
+* **SCM Provider Organization `admin` Permissions**
+  * This is only required when creating the webhook
+  * The workload cluster can still be created without the SCM webhook
```

Comment:
This is jumping the gun (we aren't in the workload cluster docs).  Just say the SCM connection can still function without a webhook, you just won't receive PR merge/close updates from the provider.